js/src/modal.js

```diff
@@ -197,7 +198,21 @@ class Modal {
   dispose() {
     $.removeData(this._element, DATA_KEY)
 
-    $(window, document, this._element, this._backdrop).off(EVENT_KEY)
+    /**
```

Comment:
Can you remove this comment please ? We don't need to keep the history in comment, we have git which handle that for us 😉